src/contacts/EditContact/EditContact.js

```diff
@@ -180,10 +182,11 @@ class EditContact extends Component {
                     <Field
                       component={MultiSelection}
                       dataOptions={categories.map(({ id }) => id)}
+                      formatter={this.categoriesFormatter}
+                      itemToString={this.itemToString}
```

Comment:
it seems we need to use the same approach in both places into the component or not

Reply:
I think it doesn’t matter here

Reply:
@AndreiKandratsenka isn't it the same approach? I see it here https://github.com/folio-org/ui-organizations/blob/9fe3ff1bf035c67edbcc4e4b108321bda1af855a/src/VendorInformation/VendorInformationForm.js#L81

Reply:
My thought was about using this.itemToString or itemToString in both places

Reply:
@AndreiShumski2 sounds good actually, there is no need to use `this, so `itemToString={itemToString}` works for both places. Could you please handle this in additional PR?